articles/concepts/circuits.md

```diff
@@ -32,7 +32,7 @@ For example, the symbol
 
 <!--- ![](.\media\2.svg) --->
 <!-- Can't find a way to easily center this... probably an extension needed:  -->
-![Symbol for a Hadamard gate acting on a single-qubit register](~/media/concepts_2.png)
+![Symbol for a Hadamard operation acting on a single-qubit register](~/media/concepts_2.png)
 
```

Comment:
Thank you so much for the edits @Bradben Just wondering what the reason is for changing "gate"s to "operation"s. Both would be correct.